packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2721,9 +2732,25 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
     // TODO(jacobr): make this structured so clients can support clicks on
     // individual entries. For example, is this an iterable with arrows as
     // separators?
-    return StringProperty(name, debugGetCreatorChain(10));
+    return StringProperty(name, debugGetCreatorChain(50));
   }
 
+  @override
+  DiagnosticsNode describeFirstUserCreatedWidgetAncestor(String name) {
+    if (!WidgetInspectorService.instance.isWidgetCreationTracked()) {
+      return StringProperty(
+        name,
+        'Enable widget creation tracking by using flutter run flag `--track-widget-creation`.',
```

Comment:
If I see this error when widget tracking is disabled, is there anything here to explain to me why I would want to do this?

It seems like we want to add something here like 
````
'Widget creation tracking is currently disabled. Enabling it would lead to improved error messages.  It can be enabled by passing `--track-widget-creation` to `flutter run` or `flutter build`'
````

Reply:
fixed the language